exporter/sumologicexporter/README.md

```diff
@@ -10,6 +10,61 @@
 [contrib]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
 <!-- end autogenerated section -->
 
+## Migration to new architecture
+
+**This exporter is subject of major changes now.**
+
+For some time we have been developing [new Sumo Logic exporter](https://github.com/SumoLogic/sumologic-otel-collector/tree/main/pkg/exporter/sumologicexporter#sumo-logic-exporter) and now we are in progress of moving it into this repository.
+
+The following options are deprecated and they won't exist in new version:
+
+- `metric_format: {carbon2, graphite}`
+- `metadata_attributes: [<regex>]`
+- `graphite_template: <template>`
+- `source_category: <template>`
+- `source_name: <template>`
+- `source_host: <template>`
```

Comment:
We're going to drop some other settings from the distro exporter very soon as well:

- `clear_logs_timestamp`
- `routing_atttribute_to_drop`
- `json_logs.add_timestamp`
- `json_logs.flatten_body`

Do you think we should mention them here as well, even though we haven't dropped them yet in the fork?

Reply:
I wouldn't mention them here, as they never existed in this repository code